gulpfile.js

```diff
@@ -1545,6 +1589,24 @@ gulp.task(
   })
 );
 
+gulp.task("integration-pre", function (done) {
+  process.env.INTEGRATIONTEST = "true";
+  done();
+});
+
+gulp.task(
+  "integrationtest",
+  gulp.series(
+    "integration-pre",
+    "testing-pre",
+    "generic",
+    "components",
```

Comment:
Are you actually using the output of the "components" build for anything here?

Reply:
No